### PR TITLE
refactor: explicit modification

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/addon-charts/components/arc-chart/arc-chart.component.ts [hunk 1]
```diff
@@ -65,32 +65,34 @@ export class TuiArcChartComponent {
     private readonly sanitizer = inject(DomSanitizer);
     private readonly arcs$ = new ReplaySubject<QueryList<ElementRef<SVGElement>>>(1);
 
+    protected initialized = false;
+
     @ViewChildren('arc')
-    set arcs(arcs: QueryList<ElementRef<SVGElement>>) {
+    public set arcs(arcs: QueryList<ElementRef<SVGElement>>) {
```
Comment (nsbarsukov) [inline projects/addon-charts/components/arc-chart/arc-chart.component.ts:71]: Could it be also `protected` ?

File: projects/addon-charts/components/arc-chart/arc-chart.component.ts [hunk 2]
```diff
@@ -117,36 +117,36 @@ export class TuiArcChartComponent {
 
     @HostBinding('style.width.rem')
     @HostBinding('style.height.rem')
-    get width(): number {
+    public get width(): number {
```
Comment (nsbarsukov) [inline projects/addon-charts/components/arc-chart/arc-chart.component.ts:120]: Could it be also `protected` ?

Comment (nsbarsukov) [reply projects/addon-charts/components/arc-chart/arc-chart.component.ts:120]: i've tried locally => demo application works
Снимок экрана 2024-02-26 в 10 57 32

File: projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts [hunk 3]
```diff
@@ -95,150 +95,150 @@ export class TuiInputCardGroupedComponent
...
         return this.value?.card || '';
     }
 
-    get expire(): string {
+    public get expire(): string {
         return this.value?.expire || '';
     }
 
-    get cvc(): string {
+    public get cvc(): string {
```
Comment (nsbarsukov) [inline projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts:168]: Why does this file contain so many public getters?

File: projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts [hunk 4]
```diff
@@ -247,19 +247,19 @@ export class TuiInputCardGroupedComponent
...
     @HostListener('keydown.esc')
-    onEsc(): void {
+    public onEsc(): void {
         this.open = false;
     }
 
     @HostListener('keydown.arrowDown.prevent', ['$event.target', '1'])
     @HostListener('keydown.arrowUp.prevent', ['$event.target', '-1'])
-    onArrow(element: HTMLElement, step: number): void {
+    public onArrow(element: HTMLElement, step: number): void {
```
Comment (nsbarsukov) [inline projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts:256]: `protected` ?

File: projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts [hunk 5]
```diff
@@ -273,7 +273,7 @@ export class TuiInputCardGroupedComponent
         element?.focus();
     }
 
-    onCardChange(card: string): void {
+    public onCardChange(card: string): void {
```
Comment (nsbarsukov) [inline projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts:276]: `protected` ?

**Pull request closed without merging**